client/src/screens/Login/Login.styled.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { styled } from '@mui/material/styles';
+import MuiPaper from '@mui/material/Paper';
+import MuiBox from '@mui/material/Box';
+import MuiTypography from '@mui/material/Typography';
+import MuiLink from '@mui/material/Link';
+import MuiSnackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
```

Comment:
could import once from `@mui/material`
for example: 
`import {Paper as MuiPaper, Box as MuiBox, ... } from '@mui/material'

Reply:
no, but currently stay it like that